Console.Calculator.oksanatk/CalculatorLibrary.cs

```diff
@@ -0,0 +1,189 @@
+﻿using Newtonsoft.Json;
+using Spectre.Console;
```

Comment:
🟢 **Spectre Console**

⭐ Very creative use of spectre to bring the console to life with colour!